sdk/iOS/src/MSPullSettings.m

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import "MSPullSettings.h"
+
+#pragma mark * MSPullSettings Implementation
+
+@implementation MSPullSettings
+
+static NSInteger const MSDefaultPageSize = 50;
+
+#pragma mark * Initializer method(s)
+
+- (instancetype)init {
```

Comment:
Yes, it sure is a bit awkward, but as this class is just a container for miscellaneous settings it will be difficult to have initializers for all properties as the property list grows. At that point it will not be possible to remove initWithPageSize (breaking change) and at the same time having an initializer for page size but not for others would seem weird. What do you think?